components/ReviewSection.tsx

```diff
@@ -0,0 +1,47 @@
+import {
+  Heading,
+  HeadingLevels,
+  Paragraph,
+  SummaryList,
+} from "lbh-frontend-react";
+import React from "react";
+import PropTypes from "../helpers/PropTypes";
+import { SectionRow } from "../helpers/useReviewSectionRows";
+
+interface Props {
+  section?: {
+    heading?: string;
+    rows?: SectionRow[];
+  };
+}
+
+export const ReviewSection: React.FunctionComponent<Props> = (props) => {
+  const { section } = props;
+  const { heading, rows } = section || {};
+
+  return (
+    <React.Fragment key={heading}>
+      <Heading level={HeadingLevels.H2}>{heading}</Heading>
+
+      {rows && rows.length ? (
+        <SummaryList
+          rows={(rows as unknown) as { key: string; value: string }[]}
```

Comment:
Make sure you note the reason for this cast in the commit message if you haven't.

Reply:
I've actually removed this, and it doesn't seem to be breaking anything - unless I've misunderstood what the purpose of it was...